x-pack/legacy/plugins/canvas/public/components/shape_picker_popover/shape_picker_popover.tsx

```diff
@@ -33,10 +34,6 @@ export const ShapePickerPopover = ({ shapes, onChange, value }: Props) => {
       {() => <ShapePicker onChange={onChange} shapes={shapes} />}
     </Popover>
   );
-};
+});
 
-ShapePickerPopover.propTypes = {
```

Comment:
Is propTypes not compatible with React.memo?

Reply:
@poffdeluxe  it's compatible, I did revert back my changes

Reply:
@poffdeluxe would you mind to review the changes once again?